README.md

```diff
@@ -39,8 +45,36 @@ loop {
 }
 ```
 
-For a more complete example, see [examples/simple.rs](https://github.com/nomic-io/abci2/blob/master/examples/simple.rs) (you can run it via `cargo run --example simple`).
+For a more complete example, see [examples/simple.rs](https://github.com/turbofish-org/abci2/blob/master/examples/simple.rs) (you can run it via `cargo run --example simple`).
 
-## Rebuild Protobuf
+abci2 is currently used by [Nomic](https://github.com/nomic-io/nomic), a blockchain powering decentralized custody of Bitcoin, built on [Orga](https://github.com/turbofish-org/orga).
```

Comment:
This is misleadimg since Nomic doesn’t depend on abci2, it just depends on orga which depends on abci2. Consumers of orga aren’t aware of or exposed to this relationship.